README.md

```diff
@@ -1,8 +1,13 @@
-# terra-featured-workspace-tests
+# TERRA FEATURED WORKSPACES TESTS
 This code is to test that Terra Featured Workspaces are working properly. It generates individual reports for each workspace, as well as a master report listing all the tested workspaces with links to their individual reports. The reports live in a google bucket and are publicly viewable.
 
 Note: Currently this code tests all workflows within a Featured Workspace, but it does not test notebooks.
 
+### Development Notes
+
+1. Tests are divided in 4 batches, there exists this parameter in code `batch_size = 18` that needs to be changed if number of featured workspaces. 
```

Comment:
I think this sentence is missing its end:
"if the number of featured workspaces **increases**"

Reply:
This can be ignored, but it's still missing a word.